5/README.md

```diff
@@ -45,23 +53,72 @@ There are many use cases for the usage of a Property:
 
 * To change the representation of an information in order to be more user friendly a Property can define a new data format for the graphical user interface to convert the raw data.
 
+* TODO: Memorized Attribute
 
-## Specification
-
-A Tango Property is a strict definition of a pair of key/value
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
-
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
 
+## Specification
 
 A Tango Property is a strict definition of a pair of key/value
 * The Property SHALL have one key, called Property Name
-* The Property SHALL have one value
+* The Property SHALL have one value, which could be empty, called Property Value
+* If the device is started upon a Tango Database, Class, Device, Attribute and Free Properties MAY be persisted into the Tango Database (RFC6)
+* If the device is started without a Tango Database, the device and class properties MAY be persisted in a file (but no support for the attribute properties) 
+* All device and class properties MAY be directly modified and accessed through the database device or its file
+* Attribute properties SHOULD be modified and accessed at any time through the device object (RFC-2)
+* All default attribute and device properties are OPTIONAL
+* The attribute, class and device properties MAY be defined in the device code and they MAY be OPTIONAL OR REQUIRED (mandatory)
+* TODO memorized attributes, system properties
+* TODO : properties history
+* TODO: error management
+* All device and class properties MUST be loaded at init command or device start-up
+* If a device property does not exist, the device MUST load the class property having the same name
+* All device and class properties MAY be modified and accessed through the database device when using a Tango Database (RFC-6)
+* A Tango device MUST manage the following default device properties: cmd_min_poll_period, min_poll_period, attr_min_poll_period, poll_ring_depth, cmd_poll_ring_depth, attr_poll_ring_depth, polled_attr, logging_target, logging_level, logging_rft (TODO: description of each one)
+* An attribute MUST manage the following default attribute properties (TODO: descriptions):
```

Comment:
It it Property RFC or Attribute RFC which shall describe things like *max_alarm*?
Yesterday I've started to work on Attribute RFC. You may check if you want, but I still need do some rework and testing before submitting a PR:
https://github.com/mliszcz/rfc/blob/fix-28-attribute-spec/4/README.md

Reply:
I propose to create multiple RFC's for all the different types of attribute properties:

1. Attribute Configuration properties
2. Attribute Event properties
3. Attribute Properties

@andygotz + @bourtemb